server/src/main/java/keywhiz/service/resources/admin/SecretsResource.java

```diff
@@ -211,7 +211,16 @@ protected SanitizedSecret retrieveSecret(@Auth User user, String name) {
   @Consumes(APPLICATION_JSON)
   public Response createSecret(@Auth User user, @Valid CreateSecretRequestV2 request) {
 
-    logger.info("User '{}' creating secret '{}'.", user, request.name());
+    {
+      String ownerPart = request.owner() == null
+          ? "with no owner"
+          : String.format("with owner '%s'", request.owner());
+      logger.info(
+          "User '{}' creating secret '{}' {}.",
+          user,
+          request.name(),
+          ownerPart);
+    }
```

Comment:
Maybe I'm overthinking it, but I didn't want a missing owner to be represented by 'null' in the output, since that looks a lot like a string containing the word null and not a null value.